nipype/utils/profiler.py

```diff
@@ -85,11 +88,13 @@ def _sample(self, cpu_interval=None):
             try:
                 with child.oneshot():
                     cpu += child.cpu_percent()
-                    mem += child.memory_info().rss
+                    mem_info = child.memory_info()
+                    rss += mem_info.rss
+                    vms += mem_info.vms
             except psutil.NoSuchProcess:
                 pass
 
-        print('%f,%f,%f' % (time(), (mem / _MB), cpu),
+        print('%f,%f,%f,%f' % (time(), rss / _MB, cpu, vms / _MB),
```

Comment:
a little nitpick here: maybe best putting cpu right after the timestamp? - sorry I didn't suggest this in the first round.

Reply:
Sure. I was trying to leave anything that depends on the old columns usable, but I agree that I'd rather have `time,cpu,rss,vms`.

Reply:
let's make the change now, better than after people start using this for real.